src/main/java/org/mule/extension/http/internal/request/HttpPollingSource.java

```diff
@@ -186,8 +216,56 @@ public void poll(PollContext<String, HttpResponseAttributes> pollContext) {
   }
 
   @Override
-  public void onRejectedItem(Result<String, HttpResponseAttributes> result, SourceCallbackContext sourceCallbackContext) {
-    LOGGER.debug("Item rejected by HTTP Polling Source in flow '{}' with result: '{}", location.getRootContainerName(),
+  public void onRejectedItem(Result<String, HttpResponseAttributes> result,
+                             SourceCallbackContext sourceCallbackContext) {
+    LOGGER.debug("Item rejected by HTTP Polling Source in flow '{}', result: '{}'", location.getRootContainerName(),
                  result.getOutput());
   }
+
+  private BindingContext buildContext(TypedValue<String> payload, HttpResponseAttributes attributes,
+                                      java.util.Optional<TypedValue<CursorStreamProvider>> item) {
```

Comment:
why the fqcn?

Reply:
Because I have also Optional of the sdk. 
In the next PR I am changing this the other way around (having the fqcn for the sdk one). I am not doing that right here to avoid rebase conflicts as much as possible